orcid-api-common/src/main/java/org/orcid/api/common/jaxb/OrcidExceptionMapper.java

```diff
@@ -155,6 +156,7 @@
         HTTP_STATUS_AND_ERROR_CODE_BY_THROWABLE_TYPE.put(OrcidDuplicatedElementException.class, new ImmutablePair<>(Response.Status.CONFLICT, 9030));
         HTTP_STATUS_AND_ERROR_CODE_BY_THROWABLE_TYPE.put(PutCodeRequiredException.class, new ImmutablePair<>(Response.Status.CONFLICT, 9031));
         HTTP_STATUS_AND_ERROR_CODE_BY_THROWABLE_TYPE.put(OrcidNotificationException.class, new ImmutablePair<>(Response.Status.CONFLICT, 9032));
+        HTTP_STATUS_AND_ERROR_CODE_BY_THROWABLE_TYPE.put(WrongOwnerException.class, new ImmutablePair<>(Response.Status.CONFLICT, 9034));
```

Comment:
@rcpeters no sir! we dont need it anymore, I just removed the exception from our code since noone use it anymore